config.sample.php

```diff
@@ -152,18 +152,6 @@ class Config
 	 */
 	public static $memcachedPort='11211';
 	
```

Comment:
Did you mean to remove the pusher stuff?

Reply:
yes, cuz that config was not ok. I made a config in the PR I created
https://github.com/adamlerer/webDiplomacy/blob/63099958d2a31d08204c2b83261d32e9e106ebde/config.sample.php#L489-L519